docs/config.json

```diff
@@ -27,6 +27,7 @@
             {
               "title": "Docker Compose",
               "slug": "/getting-started/docker-compose/"
+<<<<<<< HEAD
```

Comment:
Merge conflict?

Reply:
🤦 missed that one, thanks! Fixed in 60003c233393d3c441cb07e1db5d6de6bb9ce2a9